app/users/routes.js

```diff
@@ -2,10 +2,9 @@ var views = require('./views');
 
 
 module.exports = [
-
   {name: 'new', pattern: '/users/new', view: views.new_user},
   {name: 'list', pattern: '/users', view: views.list_users},
   {name: 'details', pattern: '/users/:id', view: views.user_details},
   {name: 'edit', pattern: '/users/:id/edit', view: views.user_edit},
-
+  // {name: 'grant_bucket_access', method: 'POST', pattern: '/users/:id/grant_bucket_access', view: views.grant_bucket_access},
```

Comment:
Remove commented out code. Is this supposed to be commented out?

Reply:
My idea was to leave an hint to the future person who was going to implement this feature.
Basically that person had simply to uncomment this and add a template, etc...without the risk of picking a name for this endpoint which was different.

Reply:
ok